CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to AWS ARC EKS
+
+Thank you for considering contributing to AWS ARC EKS! We appreciate your time and effort.
+To ensure a smooth collaboration, please take a moment to review the following guidelines.
+
+## How to Contribute
+
+1. Fork the repository to your own GitHub account.
+2. Clone the repository to your local machine.
+   ```bash
+   git clone https://github.com/<your_organization>/<your_terraform_module>.git
+   ```
+3. Create a new branch for your feature / bugfix.
+   ```bash
+   git checkout -b feature/branch_name
+   ```
+4. Make your changes and commit them.
+   ```bash
+   git commit -m "Your descriptive commit message"
+   ```
+5. Push to your forked repository.
+   ```bash
+   git push origin feature/branch_name
+   ```
+6. Open a pull request in the original repository with a clear title and description.
+   If your pull request addresses an issue, please reference the issue number in the pull request description.
+
```

Comment:
Please include this step

https://github.com/sourcefuse/terraform-aws-arc-ecs/blob/main/CONTRIBUTING.md#terraform-code-collaboration-guidelines